samples/interactive-tutorials/src/main/java/events/ImportUserEventsGcs.java

```diff
@@ -49,6 +50,7 @@ public static void main(String[] args) throws IOException, InterruptedException
     // TO CHECK ERROR HANDLING USE THE JSON WITH INVALID USER EVENT: gcsEventsObject =
     // "user_events_some_invalid.json"
 
+    CreateTestResources.main();
```

Comment:
I'm not sure if this would work though:
1) It will be expensive and slow to recreate everything for each single test.
2) If a parallel RemoveTestReourcesTest is running in between this line and the next line, there's still a chance that the tests will fail.

So unless we can prevent RemoveTestReourcesTest from being run, we got a chance for test failures. I'm thinking if it's OK for us to just simply ignore/remove CreateTestReourcesTest and RemoveTestReourcesTest as it may affect other tests, and this is just for an one-time set up only, not being used in future code samples.

Reply:
As an option, we can remove these 2 tests.
If it acceptable for coverage percentage, let's do it.

Reply:
@Neenu1995 The create and remove samples are one-time utility scripts to recreate the test assets. I agree that we don't need to run these every time, and that's not the intent of these - we want to have persistent assets in our repo.

Can these tests be removed / or annotated to ignore running? Also, will this cause any issues with code coverage?

Reply:
If you plan to have persistent assets, then I believe the CreateTestReourcesTest and RemoveTestReourcesTest is never used again. In that case, it is better to remove the code entirely.

But if you see a use case where this code may be used again, it is fine to keep it with an ignored annotation.

Reply:
Thanks all! Given that CreateTestReourcesTest and RemoveTestReourcesTest is removed, this line can be removed now.